src/shared/components/search-result/search-result.component.scss

```diff
@@ -16,9 +17,19 @@ table {
   tbody tr td {
     font-size: 16px;
     word-wrap: break-word;
+
     a {
+      float: left;
       color: $blue_table;
       background-color: $white;
+
+      &.govuk-link:focus {
```

Comment:
What are we doing to links that means the `govuk-link` class is being overridden in such a way that this focus state isn't working? We shouldn't need to have to re-override it...

Reply:
The specificity of the current styling here is greater than the generic govuk-link class. Therefore the links do style as expected. Which is why I have added this clause here

Reply:
So I see... but why are we even setting the colour and the background on links within the table? Is that a _different_ blue to the normal link blue? If so, that seems very odd.